google-cloud-pubsub/pom.xml

```diff
@@ -100,6 +100,11 @@
       <artifactId>google-http-client</artifactId>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
```

Comment:
This is currently causing the pipeline to fail, but should be fixed after #1420 is merged